ui/src/backoffice/literature/containers/LiteratureDetailPageContainer.tsx

```diff
@@ -5,35 +5,79 @@ import { connect, RootStateOrAny } from 'react-redux';
+import {
+  fetchLiteratureRecord,
+  resolveAction,
+} from '../../../actions/backoffice';
 import EmptyOrChildren from '../../../common/components/EmptyOrChildren';
 import LinkLikeButton from '../../../common/components/LinkLikeButton/LinkLikeButton';
 import LoadingOrChildren from '../../../common/components/LoadingOrChildren';
-import { BACKOFFICE_LITERATURE_SEARCH } from '../../../common/routes';
+import {
+  BACKOFFICE_LITERATURE_SEARCH,
+  LITERATURE,
+} from '../../../common/routes';
 import ContentBox from '../../../common/components/ContentBox';
 import { BACKOFFICE_LITERATURE_SEARCH_NS } from '../../../search/constants';
 import Breadcrumbs from '../../common/components/Breadcrumbs/Breadcrumbs';
 import DocumentHead from '../../../common/components/DocumentHead';
+import UnclickableTag from '../../../common/components/UnclickableTag';
+import { formatDateTime, resolveDecision } from '../../utils/utils';
+import LinkWithTargetBlank from '../../../common/components/LinkWithTargetBlank';
+import { isSuperUser } from '../../../common/authorization';
+import Abstract from '../../../literature/components/Abstract';
+import { columnsSubject } from './columnData';
+import { StatusBanner } from '../../common/components/Detail/StatusBanner';
+import { TicketsList } from '../../common/components/Detail/TicketsList';
 
 type LiteratureDetailPageContainerProps = {
   dispatch: ActionCreator<Action>;
-  record: Map<string, any>;
+  literature: Map<string, any>;
   loading: boolean;
+  actionInProgress: string | false;
+  isSuperUserLoggedIn: boolean;
 };
 
 const LiteratureDetailPageContainer = ({
   dispatch,
-  record,
+  literature,
   loading,
+  actionInProgress,
+  isSuperUserLoggedIn,
 }: LiteratureDetailPageContainerProps) => {
   const { id } = useParams<{ id: string }>();
 
   useEffect(() => {
     dispatch(fetchLiteratureRecord(id));
   }, []);
 
-  const data = record?.get('data');
+  const data = literature?.get('data');
   const title = data?.getIn(['titles', 0, 'title']);
+  const abstract = data?.getIn(['abstracts', 0]);
+  const controlNumber = data?.get('control_number');
+  const tickets =
+    literature?.get('tickets')?.size !== 0 && literature?.get('tickets');
+  const decision = literature?.getIn(['decisions', 0]) as Map<string, any>;
+  const status = literature?.get('status');
+  const inspireCategories = data?.get('inspire_categories')?.toJS();
+  const rawDateTime = data?.getIn(['acquisition_source', 'datetime']);
+
+  const formattedDateTime = formatDateTime(rawDateTime);
+  const acquisitionSourceDateTime = formattedDateTime
+    ? `${formattedDateTime.date} ${formattedDateTime.time}`
+    : undefined;
+  const acquisitionSourceSource = data?.getIn(['acquisition_source', 'source']);
+  const acquisitionSourceMethod = data?.getIn(['acquisition_source', 'method']);
+
+  const shouldDisplayDecisionsBox =
+    decision || status === 'approval' || (decision && status !== 'approval');
```

Comment:
can we simplify this check? are we going to need it elsewhere? Maybe move it to a util?

Reply:
yes we can!